src/server/pkg/storage/chunk/reader.go

```diff
@@ -18,14 +19,16 @@ type ReaderFunc func() ([]*DataRef, error)
 
 // Reader reads a set of DataRefs from chunk storage.
 type Reader struct {
-	ctx      context.Context
-	objC     obj.Client
-	dataRefs []*DataRef
-	curr     *DataRef
-	buf      *bytes.Buffer
-	r        *bytes.Reader
-	len      int64
-	f        ReaderFunc
+	ctx              context.Context
+	objC             obj.Client
+	dataRefs         []*DataRef
+	curr             *DataRef
+	buf              *bytes.Buffer
+	r                *bytes.Reader
+	len              int64
+	f                ReaderFunc
+	atSplit          func()
```

Comment:
I think `onSplit` is a slightly more standard (at least in our code base) name for a callback, would you mind changing it?

Reply:
Done.